### PR TITLE
Fix #1622: New SMN view

### LINKED ISSUE #1622: SMN
Add import and a simple view for [SMN](https://github.com/Illumina/SMNCopyNumberCaller) results. See MIP [#1290](https://github.com/Clinical-Genomics/MIP/issues/1290). The results file is a simple, hopefully self-explanatory tsv where we will want to show all values, possibly except the "raw" columns.

While SMA is considered a desired target for population screening by ACMG, we should still avoid showing unsolicited results - otherwise the case page might have been a nice place for it.

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality: SMNCopyNumber 

Reference to the SMN params: https://github.com/Illumina/SMNCopyNumberCaller

**How to test**:
1. how to install it
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
NOT READY FOR REVIEW

Meh, that was supposed to have been a draft. It is still missing something to connect the ind obj to the db store. Anyway, meetings soon so later.

**Comment by dnil** [comment]
Screenshot 2020-02-13 at 16 12 36

**Comment by dnil** [comment]
Ok, this could be ready for review as well now.
It is missing a few things that may require some more thinking/discussion, e.g. marking causative?

**Comment by henrikstranneheim** [comment]
Nice! With this draft view there is not that many steps from implementation in production.👍

**Comment by northwestwitch** [comment]
Observations while testing the PR, In random order:

- [ ] The last 2 items of the upper nav are not interactive. It would be nice to have a link that brings you back to the case page (it's not working) and for consistency the SMN CN should be dark:

image

This comes out of my ignorance, so feel free to ignore it if you think users will already know this stuff:
- [ ] This popover could be also described as `Whether sample is called as SMA (zero copy of SMN1)` to be consistent with the Illumina description of the field:

image

- [ ] Same as above, what about a popup with the text:`Whether sample is called as SMA carrier (one copy of SMN1)`
image

- [ ] Same as above, what about a popup with the text: `Copy number of SMN1 (None means no-call)`:
image

- [ ] Same as above, what about `Copy number of SMN2Δ7–8 (deletion of Exon7-8)`:
image

- [ ] Same as above, what about `CN of g.27134T>G, SNP associated with 2+0 silent carrier`:
image

**Comment by dnil** [comment]
> Looks like the demo file has values for params like SMN1_CN, SMN2_CN and so but everything is None at the individual level

Sorry, are you sure you reloaded? I'll drop db and test again, but fairly sure this worked with a reload?

**Comment by dnil** [comment]
>Looks like the demo file has values for params like SMN1_CN, SMN2_CN and so but everything >is None at the individual level
 
>Sorry, are you sure you reloaded? I'll drop db and test again, but fairly sure this worked with a >reload?

No - you were right; something went wrong on the clean load, that worked with my reload. Checking - thanks!

**Comment by dnil** [comment]
This should be ready for another go @northwestwitch. The main issue with loading from yaml (as on setup demo) has been fixed. I updated the tooltips I could agree with without actually reading the code. E.g. the tool aught not really to call SMA if SMN1=0 but SMN2 is high.

All in all, a tentatively better approach would be to introduce a form of fake region variant, possibly an SV of a new and similar kind. That would have the direct advantage of working with mark causatives etc. The flip side is that the SV or special variant page code becomes more complicated. 

I think the current one is good enough, but may keep tinkering on the special SV-track a little later.

File: scout/parse/case.py [hunk 1]
```diff
@@ -236,6 +313,21 @@ def parse_individual(sample):
                 'vcf2cytosure': str,
                 'capture_kits': list(str),
 
+                'upd_sites_bed': str,
+                'upd_regions_bed': str,
+                'rhocall_bed': str,
+                'rhocall_wig': str,
+                tiddit_coverage_wig': str,
```
[the remark this answers is not in the document]
Comment (dnil) [reply scout/parse/case.py]: :+1:

**Pull request merged**